policyengine_us/variables/gov/states/ca/cdss/state_supplement/payment_standard/ca_state_supplement_blind_amount.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ca_state_supplement_blind_amount(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "California SSI state supplement blind amount"
+    unit = USD
+    definition_period = MONTH
+    defined_for = StateCode.CA
+    reference = "https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=WIC&sectionNum=12200"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ca.cdss.state_supplement.payment_standard
+        person = spm_unit.members
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
```

Comment:
no action needed but filed for reference: https://github.com/PolicyEngine/policyengine-us/issues/5439